app/client/cli/keys.go

```diff
@@ -1,10 +1,13 @@
 package cli
 
 import (
+	"bytes"
 	"encoding/hex"
 	"fmt"
 	"github.com/pokt-network/pocket/logger"
-	"github.com/pokt-network/pocket/shared"
+	"github.com/pokt-network/pocket/shared/codec"
+	"github.com/pokt-network/pocket/shared/converters"
+	"github.com/pokt-network/pocket/utility/types"
```

Comment:
```suggestion
	utilTypes "github.com/pokt-network/pocket/utility/types"
```

And use `utilTypes` below just so its explicit